programs/witness_node/main.cpp

```diff
@@ -127,6 +103,25 @@ int main(int argc, char** argv) {
          return 0;
       }
 
+      cfg_options.add_options()
+              ("plugins", bpo::value<std::string>()->default_value(options.at("plugins").as<std::string>()),
```

Comment:
This is bad IMO, because it will take the options passed on the command line as default when writing a new config file.

Reply:
agree and fixed. thank you.